packages/@lwc/engine-core/src/framework/modules/attrs.ts

```diff
@@ -15,29 +15,32 @@ import {
 import { RendererAPI } from '../renderer';
 
 import { EmptyObject } from '../utils';
-import { VBaseElement, VStatic } from '../vnodes';
+import { VBaseElement, VStatic, VStaticPart } from '../vnodes';
 
 const ColonCharCode = 58;
 
 export function patchAttributes(
-    oldVnode: VBaseElement | null,
-    vnode: VBaseElement,
+    oldnode: VBaseElement | VStaticPart | null,
```

Comment:
I changed the variable names to `node` and `oldNode` because VStaticPart technically isn't a vnode as we [define it](https://github.com/salesforce/lwc/blob/master/packages/%40lwc/engine-core/src/framework/vnodes.ts#L23-L30).

Open to suggestions though!

Reply:
> **`oldnode:`**` VBaseElement | VStaticPart | null,`

> I changed the variable names to `node` and **`oldNode`**

🤨 🤭